programs/steward/Cargo.toml

```diff
@@ -17,10 +17,10 @@ no-log-ix-name = []
 cpi = ["no-entrypoint"]
 default = ["custom-heap"]
 custom-heap = []
-# idl-build = ["anchor-lang/idl-build"]
+idl-build = ["anchor-lang/idl-build", "no-entrypoint"]
 
 [dependencies]
-anchor-lang = "0.30.0"
+anchor-lang = { features = ["idl-build"], version = "0.30.0" }
```

Comment:
Hi, I was just passing by and saw you've enabled the `idl-build` feature for the default target. This should be avoided, as the IDL generation process is a separate build process that doesn't use Solana's build tools.

This seems harmless now other than adding a bit of unnecessary compilation, but it could very well break the compilation of your program in the future. For this reason, I added warnings for this usage in https://github.com/coral-xyz/anchor/pull/3061.

Reply:
@acheroncrypto thanks for the input, will make the change!